src/services/discord/commands/info/infoCmds.js

```diff
@@ -0,0 +1,115 @@
+const { MessageEmbed } = require('discord.js')
```

Comment:
It's normally a convention to name `.js` files with dashes, although some do underscores. Check out the [Google style guide here](https://google.github.io/styleguide/jsguide.html#source-file-basics) - I'd suggest renaming this and other files to demonstrate convention (e.g. `info-cmds.js`)